### PR TITLE
Regenerating the Internal CA for Ingress

### PR DESCRIPTION
**Pull request by ShudiLi**
@lihongan @jiajliu @xingxingxia @rioliu-rh @sunzhaohua2 Please help review the ingress part, thanks!

### CODE REVIEW

File: ci-operator/step-registry/openshift/oc/certrotation/openshift-oc-certrotation-commands.sh [hunk 1]
```diff
@@ -8,3 +8,39 @@ oc adm ocp-certificates regenerate-leaf -n openshift-config-managed secrets kube
...
+## create a config map and update the cluster-wide proxy configuration with the newly created config map
+oc create configmap custom-ca --from-file=ca-bundle.crt=$workPath/$proxyCert -n openshift-config
+oc patch proxy/cluster --type=merge --patch='{"spec":{"trustedCA":{"name":"custom-ca"}}}'
+
+## create secret and update the Ingress Controller configuration with the newly created secret
+oc create secret tls custom-secret --cert=${workPath}/${cusIngressCert} --key=${workPath}/${cusIngressKey} -n openshift-ingress
+oc patch ingresscontroller.operator default --type=merge -p '{"spec":{"defaultCertificate": {"name": "custom-secret"}}}' -n openshift-ingress-operator
+
+## Delete the router-ca secret.
+oc -n openshift-ingress-operator delete secrets/router-ca
```
Comment (lihongan) [inline ci-operator/step-registry/openshift/oc/certrotation/openshift-oc-certrotation-commands.sh:42]: see the document, need to delete the ingress operator pod to force regenerate the router-ca

**Comment by lihongan** [comment]
cc @melvinjoseph86

File: ci-operator/step-registry/openshift/oc/certrotation/openshift-oc-certrotation-commands.sh (hunk 1 above)
Comment (ShudiLi) [reply ci-operator/step-registry/openshift/oc/certrotation/openshift-oc-certrotation-commands.sh:42]: Thanks for the view, updated including checking new ingress-controller pod and new secret router-ca

**Comment by ShudiLi** [comment]
/pj-rehearse  pull-ci-openshift-oc-master-e2e-aws-certrotation

**Comment by ShudiLi** [comment]
/pj-rehearse pull-ci-openshift-oc-release-4.14-e2e-aws-certrotation

**Comment by ShudiLi** [comment]
/close
sorry, should create the PR for openshift-upgrade-qe-test-cert-rotation-commands.sh

**Comment by ShudiLi** [comment]
New https://github.com/openshift/release/pull/40789, thanks

**Pull request closed without merging**